.ci/bench_bft_sync.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+network_id=1
+min_height=200
```

Comment:
Just an idea, if the bft sync takes so much longer than client sync with the current setup, we could lower this to a height of 60 (which still surpassing the GC limit if we have strictly 2-round blocks). The smaller the benchmark, the lower the chance for noise building up in the measurement.